dist/tools/randhex/randhex.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
the file is using bashisms. I'd prefer to fix those, but a quickfix is to explixitly use bash in the shebang.

Reply:
> the file is using bashisms.

Could you point those out to me? I would rather prefer fixing them.

Reply:
> Could you point those out to me? I would rather prefer fixing them.

my mind saw some, checkbashisms doesn't agree. sorry for the noise.

Reply:
no problem.